### ISSUE TITLE
Questions regarding the SIDD dataset split

### ISSUE THREAD

**Issue by kts707** [swz30/MIRNet#54]
Hi,

Thanks for the great work!

I am a little bit confused with how the [SIDD validation set images](https://drive.google.com/drive/folders/1j5ESMU0HJGD-wU6qbEdnt569z7sM3479) are generated. Did you take the Validation .mat files from SIDD benchmark [here](https://www.eecs.yorku.ca/~kamel/sidd/benchmark.php) (as shown in the screenshot below), and then convert them to image patches as described in #12? 

In other words, the validation image patches on your Google Drive should be the same content as these .mat files, and they do not have any overlap with the actual training data? Some further clarifications on the dataset split would be super helpful. Much appreciated! 

![SIDD](https://github.com/swz30/MIRNet/assets/59402345/b285176d-a83f-48a7-9a98-00c20d1102fa)

**Comment by adityac8** [comment]
For the Validation set we extract the images from the mat files. There is no overlap between the train and val set.

**Comment by kts707** [comment]
Thanks for the quick reply! 

I also noticed that in your later work such as in MPRNet, MIRNetv2, and Restormer, you have a validation set and a test set [here](https://github.com/swz30/MIRNetv2/blob/main/Real_Denoising/download_data.py#L22-L23). It seems that the validation images are the same ones as in this repo, and the test set has the validation .mat files from SIDD website. Are they essentially the same content, except that val set are the cropped patches of the test set (.mat files)?
